test/CMakeLists.txt

```diff
@@ -230,6 +230,7 @@ endif()
 
 target_link_libraries(unit-test PRIVATE ngraph_test_util)
 target_link_libraries(unit-test PRIVATE ngraph libgtest)
+target_link_libraries(unit-test PRIVATE -Wl,--whole-archive hybrid -Wl,--no-whole-archive)
```

Comment:
Will this work on Windows or macos? Looks gcc and gnu-ld specific.

Reply:
fixed to only run on linux